### ISSUE TITLE
How do i add custom social media button with icon and link ?

### ISSUE THREAD

**Issue by jasyohuang** [konstantinmuenster/gatsby-starter-portfolio-minimal-theme#12]

**Comment by konstantinruge** [comment]
That's not possible at the moment. The following profiles can be added:
```ts
export enum SocialProfile {
    Behance = 'behance',
    GitHub = 'github',
    Medium = 'medium',
    Mail = 'mail',
    LinkedIn = 'linkedin',
    Twitter = 'twitter',
}
```

But if you want, I could add your wanted platform to the list?

**Comment by aayusranjan** [comment]
Is there any chance you can add **Hashnode** platform.

**Comment by konstantinruge** [comment]
@aayusranjan Yes, definitely. I will add it and let you know in this issue once I released a new version 👍

**Comment by konstantinruge** [comment]
@aayusranjan I just released v3.5.0 which contains an icon for Hashnode 😊
